### PR TITLE
Themes issues, ui issues

### PR DESCRIPTION
**Pull request by maxgrapps**
1) Colors issues and suggestions for all themes. Thats how they look like now:
- blockscout, poa
blockscout
- rsk
rsk
- etc
etc
- eth
eth
- xdai
xdai
- sokol
sokol
- kovan
kovan
- ropsten
ropsten
- goerli
goerli
- rinkeby
rinkeby

2) Added shadow for statistics block
Screen Shot 2019-06-06 at 17 05 00

3) Changes transition for buttons, dropdowns, tabs, etc

4) Logs search size increased, added logic for hiding pagination, font-weight issue for input
(Please, review apps/block_scout_web/lib/block_scout_web/templates/common_components/_pagination_container.html.eex . Is it correct? I am added 'disabled' for all pagination container)
Screen Shot 2019-06-06 at 17 06 28
Can we hide search if theres no logs also?

5) Increased margin between dropdown and textarea
Screen Shot 2019-06-06 at 17 08 01

6) Added current styles for heading and copy button in 'code' tab
Screen Shot 2019-06-06 at 12 50 11

7) Network list aligned to the right
Screen Shot 2019-06-06 at 17 10 04

### CODE REVIEW

File: apps/block_scout_web/assets/css/components/_btn_dropdown_line.scss [hunk 1]
```diff
@@ -9,8 +9,8 @@ $btn-dropdown-line-font: #333;
   outline: none !important;
 
   &:hover {
-    background-color: $btn-dropdown-line-color;
-    border-color: $btn-dropdown-line-color;
+    background-color: #f5f6fa;
+    border-color: #f5f6fa;
```
Comment (vbaranov) [inline apps/block_scout_web/assets/css/components/_btn_dropdown_line.scss]: Let's use var for this repeated colour?

Comment (maxgrapps) [reply apps/block_scout_web/assets/css/components/_btn_dropdown_line.scss]: @vbaranov resolved

**Comment by vbaranov** [comment]
@maxgrapps fix eslint test:
`./node_modules/.bin/eslint --fix`

**Comment by maxgrapps** [comment]
@vbaranov seems, it's okay now

**Pull request merged**